src/visualize.py

```
@@ -0,0 +1,104 @@
+
+    if len(misclassified_indices) == 0:
+        print("Gratulálok! A modell nem hibázott a validációs adatokon.")
+        return
+
+    # Véletlenszerű mintavétel az elrontottakból
+    num_to_sample = min(num_images, len(misclassified_indices))
+    selected_indices = np.random.choice(misclassified_indices, num_to_sample, replace=False)
+
+    # Ábra előkészítése (5 oszlop)
+    rows = int(np.ceil(num_to_sample / 5))
+    fig, axes = plt.subplots(rows, 5, figsize=(15, 3 * rows + 3))
```

Ruff (0.14.6)

Unpacked variable `fig` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)